js/util/metadataPropUtil.ts

```diff
@@ -2,7 +2,7 @@ import NavigationUtil from './navigationUtil';
 
 const { Platform } = require('react-native');
 
-const { version } = require('../../package.json');
+const { version } = require('../../../package.json');
```

Comment:
Added this because the structure of `dist` changed.  This breaks a unit test so we'll need to figure that out soon.